Install-Under-CentOS_LAMP.md

```diff
@@ -452,6 +452,38 @@ documentation on how to make your SELinux policy right.
    ```console
    setenforce 1
    ```
+   
+### Considerations when using Proxys in front of Cacti (Cacti 1.2.23+)
+
+In the config.php file you will have an option to allow/deny proxies based on IP or Headder
+this can be set by editing the following section of config.php 
+
+```
+ * Allow the use of Proxy IPs when searching for client
+ * IP to be used
+ *
+ * This can be set to one of the following:
+ *   - false: to use only REMOTE_ADDR
+ *   - true: to use all allowed headers (not advised)
+ *   - array of one or more the following:
```

Comment:
This is a duplicate of the other section.  We should make a small comment and link to the full section if it is in more than one place.